pkg/abi/linux/fuse.go

```diff
@@ -498,6 +545,63 @@ type FUSEWriteIn struct {
 	_ uint32
 }
 
+// SlicePayload represents a byte array which can be marshalled.
+//
+// +marshal dynamic
+type SlicePayload []byte
+
+// SizeBytes implements marshal.Marshallable.SizeBytes.
+func (s *SlicePayload) SizeBytes() int {
+	if s == nil {
+		return 0
+	}
+	return len(*s)
+}
+
+// MarshalBytes implements marshal.Marshallable.MarshalBytes.
+func (s *SlicePayload) MarshalBytes(dst []byte) []byte {
+	copy(dst, *s)
+	return dst[s.SizeBytes():]
+}
+
+// UnmarshalBytes implements marshal.Marshallable.UnmarshalBytes.
+func (s *SlicePayload) UnmarshalBytes(dst []byte) []byte {
+	panic("Unimplemented, SlicePayload is never unmarshalled")
+}
```

Comment:
Remove this and instead use [primitive.ByteSlice](https://cs.opensource.google/gvisor/gvisor/+/master:pkg/marshal/primitive/primitive.go;l=71;drc=0fd9b69d5ccd85c1958030c796470f56ec22fffa).